app/Events/User/BanRepealed.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Events\User;
+
+use App\Models\Ban;
+use App\User;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class BanRepealed
```

Comment:
Could be worth setting a base Ban event since the class properties are the same.